server/app/schema.py

```diff
@@ -119,6 +122,11 @@ class EdgeBase(graphene.ObjectType, name=f"{node._meta.name}Edge"):
 
         super(NonNullConnection, cls).__init_subclass_with_meta__(node=_node, **kwargs)
 
+    total_count = graphene.Int()
```

Comment:
required=True, then you can remove the !s after the totalCounts in frontend code